IdokladSdk/Requests/Mail/Async/IssuedDocumentPaymentConfirmationEmail.Async.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using IdokladSdk.Response;
+
+namespace IdokladSdk.Requests.Mail
+{
+    public partial class IssuedDocumentPaymentConfirmationEmail
+    {
+        /// <inheritdoc/>
+        public Task<ApiResult<bool>> SendAsync(int settings, CancellationToken cancellationToken = default)
```

Comment:
Settings?

Reply:
je to implementácia z interface IEmail, kde sa tak volá premenná, ale môžem to premenovať